Mac-App/Frontend/Frontend/ViewData/ProjectDetails.swift

```diff
@@ -9,29 +9,32 @@
 import Foundation
 
 public enum ProjectDetails {
-    // MARK: - View Data
+    // MARK: - View State
 
-    public struct Data: Equatable {
+    public struct State: Equatable {
```

Comment:
I figured using State instead of Data fits better with the Redux naming convention for `AppState` which I'm already following in the `Backend`. 

I believe this also reads better at call site.

Reply:
I think soo too 👍